src/universal_transfer_operator/__init__.py

```diff
@@ -1,6 +1,6 @@
 """An Operator that allows transfers between different datasets."""
 
-__version__ = "0.0.1dev1"
+__version__ = "0.1.0dev1"
```

Comment:
The version will be 0.1.0, right?

Reply:
Thats correct.

Reply:
Ya, okay but now we have to push another commit to release a version update.